tests/cloudwatch_handler.py

```diff
@@ -0,0 +1,26 @@
+import os
+import typing
+from enum import Enum
+
+import boto3
+
+
+class CloudwatchHandler:
+    def __init__(self):
+        self.namespace = f"dcp-dcp-test-{os.environ['DEPLOYMENT_ENV']}"
```

Comment:
`dcp-dcp-`?
If it is legit, maybe add a comment explaining why.

Reply:
oops. fixing